### DISCUSSION TITLE
[Safari] XMLHttpRequest cannot load XXX due to access control checks.

### DISCUSSION THREAD

**Discussion by facundocapua** [mercadopago/sdk-react#102] [Q&A] [answered]
Buenas tardes, me está dando este error:
![image](https://github.com/mercadopago/sdk-react/assets/1374929/35dfce7c-dd92-4196-9b80-ff2291d37b4f)

Sólo ocurre en Safari, tanto en desktop como en iOS. 

Probé actualizando el header de CSP por si ayudaba pero no ha resuelto nada.

¿Me podrían ayudar con este problema?

**Comment by icaldana** [comment]
Hola @facundocapua, buenas!
Gracias por utilizar nuestro producto!

Aún estás con ese problema? Estás probando en entorno local, cierto?

**Reply by facundocapua** [reply]
Hola @icaldana si sigo con el problema.

De momento, y como forma alternativa, estoy utilizando el init url que devuelve cuando genero la preferencia.  Pero el error de Javascript persiste.

**Reply by icaldana** [reply, marked as answer]
Bien, yo hice algunas pruebas y no pude reproducir este escenario, incluidas pruebas en Safari. Pero dejo algunos enlaces que pueden ayudar:

1. Tenemos un [ejemplo de implementación](https://github.com/mercadopago/checkout-payment-sample/tree/master/client/reactjs) que quizá ayude 
2. La [documentación oficial](https://www.mercadopago.com.br/developers/en/docs/checkout-bricks/payment-brick/advanced-features/preferences) con ejemplos de creación de preferencias, vía backend (lo recomendable)
